libraries/chain/controller.cpp

```diff
@@ -750,6 +750,24 @@ struct controller_impl {
          // else no checks needed since fork_db will be completely reset on replay anyway
       }
 
+      if (auto dm_logger = get_deep_mind_logger()) {
+         // FIXME: We should probably feed that from CMake directly somehow ...
+         fc_dlog(*dm_logger, "DEEP_MIND_VERSION 12");
+
+         fc_dlog(*dm_logger, "ABIDUMP START");
+         auto idx = db.get_index<account_index>();
+         for (auto& row : idx.indices()) {
+            if (row.abi.size() != 0) {
+               fc_dlog(*dm_logger, "ABIDUMP ABI ${block_num} ${contract} ${abi}",
```

Comment:
I'll probably move the `{block_num}` output in the `START` call, it's always the same here so it's not that useful to have it right here.

Reply:
The `{block_num}` is now printed only once when starting the ABI dump process.